### DISCUSSION TITLE
Settings not visible after upgrade/install

### DISCUSSION THREAD

**Discussion by aloftnfurl** [netalertx/NetAlertX#595] [Q&A]
After upgrading to 24.3.19 I seem to have lost visibility to my settings.  
Upgrade completed successfully and seems to be running fine except that the settings page shows just the headings...no actual settings are displayed.

![image](https://github.com/jokob-sk/Pi.Alert/assets/164072278/8e1518ab-15ac-4627-a0e3-bc84f06fa3b0)

All the settings categories appear the same as the image.

I've rebuilt the container. I've removed and done a fresh install of both the current and dev versions, but still can't see any of the settings.

Anyone ever run into something similar?  I'm out of ideas.

Thanks!
-aloft

**Comment by jokob-sk** [comment]
@aloftnfurl ,

Can you try clearing the cache (CTRL+F5) or try using a different browser?

If the above doesn't help, can you have a look at the dev console and share any errors you see in there? 

This guide might be helpful as well: https://github.com/jokob-sk/Pi.Alert/blob/main/docs/DEBUG_INVALID_JSON.md

**Reply by aloftnfurl** [reply]
@jokob-sk - 

It took an uninstall, a cache clear, and a reinstall but things are as they should be.

Appreciate the help!

-aloft
